kong/plugins/apifortress/schema.lua

```diff
@@ -0,0 +1,15 @@
+-- The plugin schema
+
+local function validateKeys(value)
+	return string.len(value) > 0
+end
+
+return {
+  fields = {
+   	apikey = { required = true, type = "string", func = validateKeys },
+		secret = { required = true, type = "string", func = validateKeys },
+		endpoint = {required = true, type = "string", func = validateKeys},
+		projectId = {required = true , type = "number", func = validateKeys},
+    threshold = {required = true, type = "number", func = validateKeys}
```

Comment:
Maybe what you'd want is `v > 0` here? I haven't yet looked at the actual functionalities entirely so I don't know yet.